cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/purgatory/ReviewStatus.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2019 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.servlet.purgatory;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+
+/**
+ * Possible status of requests in {@link Purgatory}.
```

Comment:
Maybe add explanation of state transition here.

Reply:
Transitions between different `ReviewStatus` is relevant to the user; hence, were documented in `RequestInfo` JavaDoc.